app/src/app/pages/iconography/iconography.component.html

```diff
@@ -17,5 +17,26 @@
     </div>
   </div>
 
-  <app-slider [items]="sliderItems"></app-slider>
+  <ul class="nav nav-tabs">
+    <li class="nav-item">
+      <a class="nav-link" [ngClass]="{ active: showCurrentIconographyArtworks }" data-toggle="tab" href="#current_iconography_artworks_tab">
+        <i class="mr-1 fa fa-palette"></i>
+        <ng-container>{{notation}}</ng-container>
+      </a>
+    </li>
+    <li class="nav-item"> 
```

Comment:
Make sure to hide this tab if it contains no artworks. Same applies for the original tab above.